shell/initialize/index.js

```diff
@@ -38,7 +38,9 @@ loadDirectives(Vue);
 Vue.component(NuxtChild.name, NuxtChild);
 Vue.component('NChild', NuxtChild);
 
-async function createApp(config = {}) {
+async function createApp() {
+  // eslint-disable-next-line no-undef
+  const config = nuxt.publicRuntimeConfig;
```

Comment:
I bumped my head multiple times into this and I wondered: do we know where this `nuxt` is coming from?

Reply:
Yeah, it's solely supplied by an environment variable. Instead of passing the config around, I thought it to be more clear to simply access the environment viariable when we require it.